manifests/service.pp

```diff
@@ -6,11 +6,23 @@
     fail("Use of private class ${name} by ${caller_module_name}")
   }
 
-  service { 'jenkins':
-    ensure     => $jenkins::service_ensure,
-    enable     => $jenkins::service_enable,
-    hasstatus  => true,
-    hasrestart => true,
+  case $::osfamily  {
+    'OpenBSD': {
+        service { 'jenkins':
+          ensure     => $jenkins::service_ensure,
+          enable     => $jenkins::service_enable,
+          flags      => $jenkins::service_flags,
```

Comment:
Why does this need to be it's own case?  If flags is empty, it'll be fine, iirc.